inspirehep/modules/theme/ext.py

```diff
@@ -41,6 +43,7 @@ def __init__(self, app=None, **kwargs):
         self.menu_ext = Menu()
         self.menu = None
         self.breadcrumbs = Breadcrumbs()
+        self.weblinks = self.init_weblinks_dictionary()
```

Comment:
Design: why not make this an object creation using a `KB` class that wraps the file accessing? This could become our guerrilla substitute for Invenio-Knowledge.